tutorials/plot_bem.py

```diff
@@ -0,0 +1,45 @@
+"""
+==================================
+Compute forward model and plot BEM
+==================================
+
+MNE-HCP makes it easy to construct inverse models (for source localisation) from HCP data.
+Here, we compute the full "forward stack" and graphically check if the BEM solution is well-behaved.
+"""
+# Author: Jona Sassenhagen
+# License: BSD 3 clause
+
+import hcp
+import mne
+
+hcp_path = '/Volumes/backup/'  # XXX: fix
+subject = '100307'
+subjects_dir = hcp_path + 'subjects'
+
+anatomy_params = dict(
+    subject=subject, anatomy_path=subjects_dir,
+    hcp_path=hcp_path + 'HCP',
+    recordings_path=hcp_path + 'hcp-meg'
+    )
+
+########################################################
+# First, create the required anatomical files
+
+hcp.make_mne_anatomy(**anatomy_params)
+
```

Comment:
can we drop this and just add a comment that it is expected to have this set up. We don't want examples that write to the users file system. We will add a separate example that explains this step.